tuning/tune.py

```diff
@@ -29,29 +33,26 @@ class Configuration:
     waves_per_eu: int
 
 
+class DispatchKind(Enum):
+    conv = 1
+    mmt = 2
+    contraction = 3
+    batch_mmt = 4
```

Comment:
This one is batch_matmul, not batch_mmt. These are different operations: https://mlir.llvm.org/docs/Dialects/Linalg/#linalgbatch_matmul-linalgbatchmatmulop whereas mmt stands for matmul_transpose_b: https://mlir.llvm.org/docs/Dialects/Linalg/#linalgbatch_matmul_transpose_b-linalgbatchmatmultransposebop
```suggestion
    batch_matmul = 4
```